source/data-formats/time-series.txt

```diff
@@ -0,0 +1,200 @@
+method:
+
+- Name of the new collection to create
+- ``timeseries`` argument 
+
+The ``timeseries`` argument is of type ``dict``. It contains the following fields:
+
+- ``timeField``
+- ``metaField``
+- ``granularity``
+- ``bucketMaxSpanSeconds``
+- ``bucketRoundingSeconds``
+
+See :manual:`Command Fields </reference/command/create/#command-fields>`
+to learn more about these fields.
+
+The following example shows how to create a time series collection using the 
+{+driver-short+}.
+
+Example
+~~~~~~~
+
+This example creates a time series collection named ``october2024`` with the
+``timeField`` option set to ``"timestamp"``.
+
+.. code-block:: python
+
+    database = client.get_database("weather")
+
+    time_series_options = {
+        "timeField": "timestamp"
+    }
+
+    database.create_collection("october2024", timeseries=time_series_options)
+
+To check if you successfully created the collection, you can print out a list of
+all collections in the database:
+
+.. io-code-block::
+    :copyable: true
+
+    .. input::
+        :language: python
+        
+        collection_list = database.list_collections()
+
+        for c in collection_list:
+            print(c)
+
+    .. output::
+        :language: json
+
+        {
+            "name": "october2024", 
+            "type": "timeseries", 
+            "options": {
+                "timeseries":   {
+                    "timeField": "timestamp",
+                    "granularity": "seconds", 
+                    "bucketMaxSpanSeconds": 3600
+                }
+            }, 
+            "info": {
+                "readOnly": False
+            }
+        }
+        ...
+
+.. _pymongo-time-series-write:
+
+Store Time Series Data
+----------------------
+
+You can insert data into a time series collection by using the ``insert_one()``
+or ``insert_many()`` methods and specifying the measurement, timestamp, and 
+metadata in each inserted document.
+
+To learn more about inserting documents, see :ref:`pymongo-write-insert`.
+
+Example
+~~~~~~~
+
+This example inserts New York City temperature data into the ``october2024``
+time series collection created in :ref:`pymongo-time-series-create`. Each
+document contains the following fields:
+
+- ``temperature``, which stores temperature measurements in degrees Fahrenheit
+- ``location``, which stores location metadata
+- ``timestamp``, which stores the measurement timestamp
+
+.. code-block:: python
+
+    from datetime import datetime
+
+    collection = database["october2024"]
+
+    document_list = [
+        { "temperature": 77, "location": "New York City", "timestamp": datetime(2024, 10, 22, 6, 0, 0) },
+        { "temperature": 74, "location": "New York City", "timestamp": datetime(2024, 10, 23, 6, 0, 0) }
+    ]
+
+    collection.insert_many(document_list)
+
+.. tip:: Formatting Dates and Times
+
+    See :ref:`pymongo-dates-times` to learn more about using ``datetime``
+    objects in {+driver-short+}.
```

Comment:
S: The [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) suggests that we state the purpose of the link before we give the link.  
```suggestion
    To learn more about using ``datetime`` objects in {+driver-short+}, 
    see :ref:`pymongo-dates-times`.
```